### DISCUSSION TITLE
How do you automatically add those headers

### DISCUSSION THREAD

**Discussion by PoIygon** [ScratchAddons/website#155] [General] [answered]
I would really like to know how to do it because I’m currently making my website from **raw HTML ** on GitHub. Wondering how I can do it.

btw this is for [when flag clicked](whenflagclicked.org)

**Comment by CST1229** [comment]
The issue templates where they automatically add text? I think Googling will help.

**Comment by retronbv** [comment]
> I would really like to know how to do it because I’m currently making my website from **raw HTML ** on GitHub. Wondering how I can do it.
> 
> 
> 
> btw this is for [when flag clicked](whenflagclicked.org)

Please don't ask questions for WhenFlagClicked here. Ask at [their repo](github.com/FlagClicked/FlagClicked)

**Comment by cobaltt7** [comment, marked as answer]
We use Hugo. Look it up.
